main/procps/APKBUILD

```diff
@@ -60,6 +60,12 @@ package() {
 	# These binaries are identical.
 	ln -sf pgrep "$pkgdir"/bin/pkill
 
+	# Override BusyBox symlinks
+	mkdir -p "$pkgdir"/usr/bin
+	for i in free pgrep pkill pmap pwdx top uptime; do
```

Comment:
Just for the next time, `i` should be declared as `local`.

Reply:
Thank you, I completely forgot about this.  I will add that to my notes.